sdk/core/core-tracing/src/tracers/opencensus/openCensusSpanWrapper.ts

```diff
@@ -40,14 +52,26 @@ export class OpenCensusSpanWrapper implements Span {
    * @param name - The name of the Span
    * @param options - Options for the Span
    */
-  constructor(tracer: OpenCensusTracerWrapper, name: string, options?: SpanOptions);
+  constructor(
```

Comment:
Since the release of OpenTelemetry has been dragged out so long, I'm not convinced we get a ton of benefit from providing the OpenCensus wrappers. It feels like it'd be not fun to support in `core-tracing` after it goes GA, so maybe we should get rid of it now? We could always bring it back as some other, optional package (or even just a code sample.)

What do you think @ramya-rao-a ? Can we ditch this now? As far as I know, nobody's ever used it.